cli.js

```diff
@@ -67,6 +68,34 @@ if (['link', 'connect', 'reset'].includes(cmd)) {
     }
 }
 
+let packageManager;
+
+if (hasYarn(cwd)) {
+    packageManager = {
+        add: modules => execa('yarn', ['add', ...modules]),
+        addDev: modules => execa('yarn', ['add', '--dev', ...modules]),
+        pack: (packFile, depPath) =>
+            execa('yarn', ['pack', '--filename', packFile], { cwd: depPath })
+    };
+} else {
+    packageManager = {
+        add: modules => execa('npm', ['install', ...modules]),
+        addDev: modules => execa('npm', ['install', '--save-dev', ...modules]),
+        pack: async (packFile, depPath) => {
+            const parsedPath = path.parse(packFile);
+            const filename = parsedPath.base;
+            // The filename npm creates doesn't have a timestamp
+            const packName = filename.substring(0, filename.lastIndexOf('-')) + '.tgz';
+            const cacheDir = parsedPath.dir;
+
+            await execa('npm', ['pack', depPath], { cwd: cacheDir });
+            fs.renameSync(
```

Comment:
whats this for ?

Reply:
never mind got it

Reply:
So that it also has the timestamp attached to the name. `npm pack` uses just the package name + version.